cloud/azure/app-services/detectors-app_services.tf

```diff
@@ -0,0 +1,191 @@
+		C = data('Requests', filter=filter('resource_type', 'Microsoft.Web/sites') and filter('is_Azure_Function', 'false') and ${module.filter-tags.filter_custom})${var.http_success_status_rate_aggregation_function}
+		signal = (((A+B)/C)*100).${var.http_success_status_rate_transformation_function}(over='${var.http_success_status_rate_transformation_window}').publish('signal')
+		aperiodic.above_or_below_detector(signal, ${var.http_success_status_rate_threshold_critical}, 'below', lasting('${var.http_success_status_rate_aperiodic_duration}', ${var.http_success_status_rate_aperiodic_percentage})).publish('CRIT')
+		aperiodic.range_detector(signal, ${var.http_success_status_rate_threshold_critical}, ${var.http_success_status_rate_threshold_warning}, 'within_range', lasting('${var.http_success_status_rate_aperiodic_duration}', ${var.http_success_status_rate_aperiodic_percentage}), lower_strict=False).publish('WARN')
+	EOF
+
+  rule {
+    description           = "is too low < ${var.http_success_status_rate_threshold_critical}"
+    severity              = "Critical"
+    detect_label          = "CRIT"
+    disabled              = coalesce(var.http_success_status_rate_disabled_critical, var.http_success_status_rate_disabled, var.detectors_disabled)
+    notifications         = coalescelist(var.http_success_status_rate_notifications_critical, var.http_success_status_rate_notifications, var.notifications)
+    parameterized_subject = "[{{ruleSeverity}}]{{{detectorName}}} {{{readableRule}}} ({{inputs.signal.value}}) on {{{dimensions}}}"
+  }
+
+  rule {
+    description           = "is too low < ${var.http_success_status_rate_threshold_warning}"
+    severity              = "Warning"
+    detect_label          = "WARN"
+    disabled              = coalesce(var.http_success_status_rate_disabled_warning, var.http_success_status_rate_disabled, var.detectors_disabled)
+    notifications         = coalescelist(var.http_success_status_rate_notifications_warning, var.http_success_status_rate_notifications, var.notifications)
+    parameterized_subject = "[{{ruleSeverity}}]{{{detectorName}}} {{{readableRule}}} ({{inputs.signal.value}}) on {{{dimensions}}}"
+  }
+}
+
+resource "signalfx_detector" "status" {
+  name = "${join("", formatlist("[%s]", var.prefixes))}[${var.environment}] Azure App Service"
+
+  program_text = <<-EOF
+		signal = data('HealthCheckStatus', filter=filter('resource_type', 'Microsoft.Web/sites') and filter('is_Azure_Function', 'false') and filter('primary_aggregation_type', 'true') and ${module.filter-tags.filter_custom})${var.status_aggregation_function}.${var.status_transformation_function}(over='${var.status_transformation_window}').publish('signal')
+		detect(when(signal < ${var.status_threshold_critical})).publish('CRIT')
```

Comment:
hum, as discused with @xp-1000 by default this option is not enabled on App Service, which means `HealthCheckStatus` might always be at 0% without any issues on the resource.

Reply:
@Shr3ps I do nat have enough documentation nor sample about this, I'll remove it.